packages/react/fast-components-styles-msft/src/utilities/color/accent-fill-gradient.ts

```diff
@@ -0,0 +1,55 @@
+import { DesignSystem } from "src";
+import { ColorHSL, hslToRGB, parseColor, rgbToHSL } from "@microsoft/fast-colors";
+import { format } from "@microsoft/fast-jss-utilities";
+import {
+    FillSwatchFamily,
+    Swatch,
+    SwatchFamily,
+    SwatchFamilyResolver,
+    swatchFamilyToSwatchRecipeFactory,
+    SwatchFamilyType,
+    SwatchRecipe,
+    SwatchResolver,
+} from "./common";
+import { accentFill } from ".";
+
+function gradientSwatchRecipe<T extends SwatchFamily>(
+    type: keyof T,
+    callback: SwatchFamilyResolver<T>
+): SwatchResolver {
+    return (designSystem: DesignSystem): Swatch => {
+        const baseRecipe: SwatchRecipe = swatchFamilyToSwatchRecipeFactory<T>(
+            type,
+            callback
+        );
+        const color: Swatch = baseRecipe(designSystem);
+        const colorHsl: ColorHSL = rgbToHSL(parseColor(color));
+        let hShifted: number = colorHsl.h + 28;
+        if (hShifted > 360) {
+            hShifted -= 360;
+        }
```

Comment:
Can this be:
```
const maxColorH: number = colorHsl.h > 332
    ? colorHsl.h - 332
    : colorHsl.h + 28;
```

I think (if I understand this correctly) that we should treat the linear gradient as having a range and that you are defining the `min` and `max` values of that range so the naming should follow that logic.

Reply:
360 is significant because it's the hue range for colors. 28 is the variable here, and 332 is just 360 - 28. I'd actually prefer this logic be in the color package, as hue is often thought of as a repeating circle, especially for wrapping like this. We could update this here, but 332 is not significant, so I wouldn't want to hard-code that. Actually, 28 could possibly be a design system property if building this for wider use.